snuba/settings/__init__.py

```diff
@@ -81,6 +81,7 @@
     {
         "host": os.environ.get("CLICKHOUSE_HOST", "127.0.0.1"),
         "port": int(os.environ.get("CLICKHOUSE_PORT", 9000)),
+        "max_connections": int(os.environ.get("CLICKHOUSE_MAX_CONNECTIONS", 100)),
```

Comment:
where did this number come from?

Reply:
Just a proposal. I have no information about how many requests snuba can process simultaneously. 

I don’t think that more than 100 connections to the clickhouse will be required for one api replica; with such values, it makes sense to scale horizontally rather than vertically.

Reply:
I would leave this as `1` (which is what it is now).

Reply:
As far as a few of us can tell, this isn't working: https://github.com/sentry-kubernetes/charts/pull/1175

Reply:
Can someone show me where the code is that actually respects/utilizes this setting?